spec.html

```diff
@@ -5819,46 +5781,54 @@ <h1>PrivateFieldAdd ( _P_, _O_, _value_ )</h1>
+        1. Append PrivateElement { [[Key]]: _P_, [[Kind]]: ~field~, [[Value]]: _value_ } to _O_.[[PrivateElements]].
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-privatemethodadd" aoid="PrivateMethodAdd">
+      <h1>PrivateMethodAdd ( _method_, _O_ )</h1>
+      <emu-alg>
+        1. Assert: _method_ is a PrivateElement Record.
+        1. Assert: _method_.[[Kind]] is either ~method~ or ~accessor~.
+        1. Let _entry_ be ! PrivateElementFind(_method_.[[Key]], _O_).
         1. If _entry_ is not ~empty~, throw a *TypeError* exception.
-        1. Append PrivateField { [[PrivateFieldName]]: _P_, [[PrivateFieldValue]]: _value_ } to _O_.[[PrivateFields]].
+        1. Append _method_ to _O_.[[PrivateElements]].
+        1. NOTE: the values for private methods and accessors are shared across instances. This step does not create a new copy of the method or accessor.
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-privatefieldget" aoid="PrivateFieldGet">
-      <h1>PrivateFieldGet ( _P_, _O_ )</h1>
+    <emu-clause id="sec-privateget" aoid="PrivateGet">
+      <h1>PrivateGet ( _P_, _O_ )</h1>
       <emu-alg>
         1. Assert: _P_ is a Private Name.
         1. Assert: Type(_O_) is Object.
-        1. If _P_.[[Kind]] is ~field~, then
-          1. Let _entry_ be ! PrivateFieldFind(_P_, _O_).
-          1. If _entry_ is ~empty~, throw a *TypeError* exception.
-          1. Return _entry_.[[PrivateFieldValue]].
-        1. Perform ? PrivateBrandCheck(_O_, _P_).
-        1. If _P_.[[Kind]] is ~method~, then
-          1. Return _P_.[[Value]].
+        1. Let _entry_ be ! PrivateElementFind(_P_, _O_).
+        1. If _entry_ is ~empty~, throw a *TypeError* exception.
+        1. If _entry_.[[Kind]] is ~field~ or ~method~, then
+          1. Return _entry_.[[Value]].
         1. Assert: _P_.[[Kind]] is ~accessor~.
-        1. If _P_ does not have a [[Get]] field, throw a *TypeError* exception.
+        1. If _P_.[[Get]] is *undefined*, throw a *TypeError* exception.
         1. Let _getter_ be _P_.[[Get]].
         1. Return ? Call(_getter_, _O_).
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-privatefieldset" aoid="PrivateFieldSet">
-      <h1>PrivateFieldSet ( _P_, _O_, _value_ )</h1>
+    <emu-clause id="sec-privateset" aoid="PrivateSet">
+      <h1>PrivateSet ( _P_, _O_, _value_ )</h1>
       <emu-alg>
         1. Assert: _P_ is a Private Name.
         1. Assert: Type(_O_) is Object.
-        1. If _P_.[[Kind]] is ~field~, then
-          1. Let _entry_ be ! PrivateFieldFind(_P_, _O_).
-          1. If _entry_ is ~empty~, throw a *TypeError* exception.
-          1. Set _entry_.[[PrivateFieldValue]] to _value_.
+        1. Let _entry_ be ! PrivateElementFind(_P_, _O_).
+        1. If _entry_ is ~empty~, throw a *TypeError* exception.
+        1. If _entry_.[[Kind]] is ~field~, then
+          1. Set _entry_.[[Value]] to _value_.
           1. Return.
```

Comment:
rather than the return here, maybe make this an "else", and indent the rest of the AO?

Reply:
Yeah, that's probably a good change, though again something to address outside this PR.